rclpy/actions/minimal_action_client/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup
+
+package_name = 'examples_rclpy_minimal_action_client'
+
+setup(
+    name=package_name,
+    version='0.6.1',
+    packages=[],
+    py_modules=[
+        'client',
```

Comment:
I think the problem is here. Both the service client and this are installing top level modules `client.py`. This should be changed to install a package instead to avoid conflicts. I'll make a PR for the other rclpy examples.

Reply:
Updated to install into package rather than top-level modules.